512-byte-vm.asm

```diff
@@ -103,6 +112,10 @@ encoded_text:
     DB 1,'Hello World is ok...',10,13   ; the indispensable hello world text
     DB 1,0                              ; end of normal texts
 
+serial_console_message:
+    DB 'Hello World!', 0
```

Comment:
@lveyde is it possible to output the same text as on the console?

Reply:
It's certainly possible, but please note that the data format is null terminated string.

Reply:
So is the other text, take a look at line 113.

Reply:
Yes, it's done on purpose - so that you could easily distinguish the text.
But if you prefer for some reason both messages to be same then I can change that.